packages/react-composites/src/composites/common/utils.tsx

```diff
@@ -48,3 +50,47 @@ export const useMinMaxDragPosition = (modalLayerHostId: string, rtl?: boolean):
 
   return { minDragPosition: minDragPosition, maxDragPosition: maxDragPosition };
 };
+
+/**
+ * @private
+ *  hidden button to set first tab keypress focus on a specific grouping.
+ *  On mount, button is autofocused then immediately hidden
+ */
+export const HiddenFocusStartPoint = (): JSX.Element => {
+  const [isMounted, setIsMounted] = useState(false);
+  useEffect(() => {
+    setIsMounted(true);
+  }, []);
+  return (
+    <DefaultButton
+      autoFocus
+      ariaHidden={true}
+      styles={isMounted ? invisibleHiddenFocusStartPoint : hiddenFocusStartPointStyles}
+      tabIndex={-1}
+    />
+  );
+};
```

Comment:
Since this is just a normal component, might be worthy to put it into a single file (even this is just util component, we even have TypeAssertion.tsx file under common/component, so this would be 100% good)

Reply:
Okay! Let's do that!
Pushed the new changes, HiddenFocusStartPoint is now in it's own file